src/nunit.analyzers.tests/DiagnosticSuppressors/NonNullableFieldOrPropertyIsUninitializedSuppressorTests.cs

```diff
@@ -123,6 +123,51 @@ public void Test()
             RoslynAssert.Suppressed(suppressor, testCode);
         }
 
+        [Test]
+        public void FieldAssignedInTryFinally()
```

Comment:
Should we rename this to `FieldAssignedInTry()` and then also have a test that tests when the assignment is in the `finally` clause?

Reply:
I have instead simplified the test and have fields assigned in both the `try` and the `finally` clauses.